easybuild/easyconfigs/p/PrgEnv-gnu/PrgEnv-gnu-tsa-19.2.eb

```diff
@@ -15,7 +15,7 @@ description = """
 
 toolchain = SYSTEM
 
-local_path = '/apps/arolla/UES/jenkins/RH7.6/%s/%s/easybuild/modules/all' % (local_prgenv, version)
+local_path = '/apps/arolla/UES/jenkins/RH7.7/%s/%s/easybuild/modules/all' % (local_prgenv, version)
```

Comment:
same here, are you sure this has no side-effects in the current deployed system?

Reply:
This one was reverted to it's original and a new module (with suffix RH7.7) was added.